lte/gateway/deploy/roles/trfserver/files/traffic_server.py

```diff
@@ -522,8 +522,14 @@ def __init__(self, server, instances):
     def _get_macs(self):
         ''' Retrieves the MAC addresses of the associated test servers, based
         on the information of the instances '''
+        dev = 'eth2'
         ip = pyroute2.IPRoute()
-        mac = ip.link('get', index=ip.link_lookup(ifname='eth2')[0])[0] \
+        # incase of missing eth2, use eth3.
+        # This check is done for compatibility for namespace based trf-serve
```

Comment:
so is `eth2` used when server is run on magma_trfserver and `eth3` when server is run on `magma_test`?

If so, can me note this here just in case this needs to be changed in the future?